piyushgarg-dev/piyushgargdev-nextjs
# fix: Text alignment in Service section

## What does this PR do?

Fixes the text alignment in the Service section. Earlier the alignment of "Popular" was not correct which has been changed.

Fixes #591

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
1. Go to Youtube Stats section
2. Check the alignment of the sentence "Popular uploads from my youtube channel".

## Mandatory Tasks


## Checklist
